core/server/api/v2/utils/serializers/input/utils/mongo.js

```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash');
```

Comment:
Think this module doesn't have to live within the core and can be moved into mongo-utils. It's rather generic and is close by concept to what mongo-utils `expandFilters` function does. Wdyt @allouis? Would also be easier to cover this small piece with a test there :wink:

Reply:
I was thinking the same thing at first, but it felt too specific for mongo-utils :man_shrugging: I'm not too fussed either way - can move it if you like though!